.github/workflows/pkgdown.yaml

```diff
@@ -0,0 +1,49 @@
+on:
+  push:
+    branches: master
+
+name: pkgdown
+
+jobs:
+  pkgdown:
+    runs-on: macOS-latest
+    env:
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
@RLesur, we need to setup a Github Token for the deployment

Reply:
Not needed in fact... 
https://help.github.com/en/actions/configuring-and-managing-workflows/authenticating-with-the-github_token